lib/gui/pages/main/controllers/flash.js

```diff
@@ -111,9 +111,12 @@ module.exports = function(
       return 'Flash!';
     }
 
-    if (flashState.percentage === 0 && !flashState.speed) {
+    const PERCENTAGE_MINIMUM = 0;
```

Comment:
Yeah, but where? `lib/shared/percentage.js`? :P I guess we can start a file with constants in `lib/shared/constants.js`. What do you think?